services/common/src/components/comments/CommentEditor.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { ReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
+import CommentEditor from "./CommentEditor";
+
+describe("CommentEditor", () => {
```

Comment:
This one needs similar initialValues so that we get a useful snapshot out of it. So long as the addCommentPermission is what's in that userAccessData state it should work.